sirepo/auth/__init__.py

```diff
@@ -531,7 +542,9 @@ def reset_state(self):
 
     def unchecked_get_user(self, uid):
         with sirepo.util.THREAD_LOCK:
-            u = sirepo.auth_db.UserRegistration.search_by(uid=uid)
+            u = self.qcall.auth_db.model("UserRegistration").unchecked_search_by(
```

Comment:
Really nice to be moving away from globals

Reply:
Yup